auth_admin_passkey/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2013-Today GRAP (http://www.grap.coop)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Authentification - System Administrator Passkey",
+    "summary": "Allows system administrator to authenticate with any account",
+    "version": "13.0.1.0.0",
+    "category": "base",
+    "author": "GRAP,Odoo Community Association (OCA)",
+    "website": "https://www.github.com/OCA/server-auth",
```

Comment:
@bizzappdev Sorry, we have another problem here with the `www.` that should be removed. Could you do that small modification, squash commits, and then we can merge,.